datahub-classify/src/datahub_classify/utils.py

```diff
@@ -0,0 +1,308 @@
+        cos_sim = 0
+    return cos_sim
+
+
+def read_glove_vector(glove_vector: str) -> dict:
+    with open(glove_vector, "r", encoding="UTF-8") as f:
+        word_to_vec_map = {}
+        for line in f:
+            w_line = line.split()
+            curr_word = w_line[0]
+            word_to_vec_map[curr_word] = np.array(w_line[1:], dtype=np.float64)
+    return word_to_vec_map
+
+
+# TODO:
+def get_fuzzy_score(
+    text_1: str,
+    text_2: str,
+    text_type: str,
+    text_1_words: List[str] = [],
+    text_2_words: List[str] = [],
+) -> Optional[float]:
+    try:
+        fuzzy_match_score = fuzz.token_set_ratio(text_1, text_2) / 100
+        if fuzzy_match_score <= 0.5:
+            fuzzy_match_score = 0.8 * fuzzy_match_score
+        max_fuzz_score = 0
+        if text_type == "name":
+            if len(text_1_words) == 1 or len(text_2_words) == 1:
+                for word_1 in text_1_words:
+                    for word_2 in text_2_words:
+                        fuzz_score = fuzz.token_set_ratio(word_1, word_2) / 100
+                        if fuzz_score > max_fuzz_score:
+                            max_fuzz_score = fuzz_score
+        fuzzy_match_score = np.maximum(fuzzy_match_score, max_fuzz_score)
+    except Exception as e:
+        logger.error(f"Fuzzy Score Not calculated {e}")
+        fuzzy_match_score = None
+    return fuzzy_match_score
+
+
+# TODO:
+def get_embedding_score(
+    text_1_emb: List[TextEmbeddings],
+    text_2_emb: List[TextEmbeddings],
+    text_1_words: List[str],
+    text_2_words: List[str],
+    word_to_vec_map: dict,
+) -> Optional[float]:
+    emb_1 = None
+    emb_2 = None
+    try:
+        if len(text_1_words) == 1 and len(text_2_words) == 1:
+            glove_emb_1 = word_to_vec_map.get(text_1_words[0], None)
+            glove_emb_2 = word_to_vec_map.get(text_2_words[0], None)
+
+            if glove_emb_1 is not None and glove_emb_2 is not None:
+                emb_1 = glove_emb_1
+                emb_2 = glove_emb_2
+            else:
+                for text_emb in text_1_emb:
+                    if text_emb.emb_type == "sentence_transformer":
+                        emb_1 = text_emb.embedding
+                        break
+                for text_emb in text_2_emb:
+                    if text_emb.emb_type == "sentence_transformer":
+                        emb_2 = text_emb.embedding
+                        break
+        else:
+            for text_emb in text_1_emb:
+                if text_emb.emb_type == "sentence_transformer":
+                    emb_1 = text_emb.embedding
+                    break
+            for text_emb in text_2_emb:
+                if text_emb.emb_type == "sentence_transformer":
+                    emb_2 = text_emb.embedding
+                    break
+        if emb_1 is None or emb_2 is None:
+            raise Exception("Embeddings Not Found!!!")
+        emb_match_score = cosine_similarity_score(emb_1, emb_2)
```

Comment:
return immediately

Reply:
are you expecting to return emb_match_score just after the calculation of cosine_similarity_score in the try block?

Reply:
if you're not modifying the `emb_match_score` anymore in the function, just return it immediately

then the thing at the end can just be `return None`

Reply:
Modified the code to return immediately